client/state/ui/reducer.js

```diff
@@ -74,6 +75,13 @@ export const isPreviewShowing = createReducer( false, {
 		isShowing !== undefined ? isShowing : state,
 } );
 
+// FIXME: We can't detect window size on the server, so until we have more intelligent detection,
+// use 769, which is just above the general maximum mobile screen width.
+export const viewportWidth = createReducer( 769, {
+	[ VIEWPORT_WIDTH_SET ]: ( state, { width } ) =>
+		! isNaN( width ) ? width : state,
+} );
```

Comment:
I tend to use `createReducer` even for small reducers, as I like its expressiveness and I like the idea of us converging towards common patterns across Calypso. That said, I also can't disagree with the argument that a simple function is better here. Just wanted to leave both views on the table. It's up to Marek to decide. :]